docs/Product-Categories/WisLink/RAK2247/Quickstart/README.md

```diff
@@ -133,6 +156,28 @@ sudo ./lora_pkt_fwd
  If you added the additional lines in step 5 it will execute every time on boot.
 :::
 
-8. The regional parameter configurations for all the supported regions are located in the folder `</opt/packet_forwarder/lora_pkt_fwd/global_conf>`. In case you need to adjust the region frequency band for example, do so before running the process (**EU868 is the default**)
-
-**Congratulations!** :tada: you should now see your Gateway in TTN!
\ No newline at end of file
+10. To check if it is working, run `sudo systemctl start ttn-gateway.service` to start the service and check its status `service ttn-gateway status`. You should see something like the box bellow. Be aware that it may take some minutes to see your gateway as connected in TTN's console.
```

Comment:
```suggestion
10. To check if it is working, run `sudo systemctl start ttn-gateway.service` to start the service and check its status `service ttn-gateway status`. You should see something like the box below. Be aware that it may take some minutes to see your gateway as connected in TTN's console.
```